components/fxa-client/src/lib.rs

```diff
@@ -68,6 +68,7 @@ pub struct FirefoxAccount {
     flow_store: HashMap<String, OAuthFlow>,
     attached_clients_cache: Option<CachedResponse<Vec<http_client::GetAttachedClientResponse>>>,
     devices_cache: Option<CachedResponse<Vec<http_client::GetDeviceResponse>>>,
+    authorization_cache: Option<oauth::AuthorizationCache>,
```

Comment:
Consider implementing `Default` and initializing this field via `Default::default()`, rather than making it an `Option` and initializing to `None`. I can't think of a situation where we'd want to have a `None` in here on purpose (although as always, am open to suggestion!).